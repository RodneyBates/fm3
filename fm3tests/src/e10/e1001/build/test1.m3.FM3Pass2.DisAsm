 ItkEOF()
 ItkLeftEnd()
 ItkModuleLt(L1(16_1),[11,0])
 ItkDeclScopeLt(L1(16_1))
 ItkOpenScopeLt(L1(16_1))
 ItkOpenDeclListLt(L2(16_2),[20,2])
 ItkProcDeclLt(Id9"P",[20,2])
 ItkDeclNo(D12,[20,12])
 ItkDeclScopeLt(L2(16_2))
 ItkSignatureProperLt([20,14])
 ItkFormalsListLt(L1(16_1),[20,16])
 ItkFormalsListRt(L1(16_1),[20,16])
 ItkResultTypeAbsent([21,0])
 ItkRaisesSetLt(L0(16_0),[21,0])
 ItkRaisesSetRt(L0(16_0),[21,0])
 ItkSignatureProperRt([20,14])
 ItkDeclScopeRt(L2(16_2))
 ItkProcDeclRt(Id9"P",[21,0])
 ItkOpenDeclListSep(L1(16_1),[20,2])
 ItkProcDeclLt(Id9"P",[21,2])
 ItkDeclScopeLt(L3(16_3))
 ItkSignatureProperLt([21,14])
 ItkFormalsListLt(L1(16_1),[21,16])
 ItkVARFormalLt([21,16])
 ItkVARFormalIdListLt(L1(16_1),[21,20])
 ItkDeclNo(D7,[21,20])
 ItkVARFormalIdListRt(L1(16_1),[21,20])
 ItkVARFormalType([21,23])
 ItkVARFormalValue([21,27])
 ItkDeclValAbsent([21,27])
 ItkVARFormalRt([21,16])
 ItkFormalsListRt(L1(16_1),[21,16])
 ItkResultTypeAbsent([23,0])
 ItkRaisesSetLt(L0(16_0),[23,0])
 ItkRaisesSetRt(L0(16_0),[23,0])
 ItkSignatureProperRt([21,14])
 ItkDeclScopeRt(L3(16_3))
 ItkProcDeclRt(Id9"P",[23,0])
 ItkOpenDeclListSep(L1(16_1),[20,2])
 ItkProcDeclLt(Id9"P",[23,2])
 ItkDeclScopeLt(L4(16_4))
 ItkSignatureProperLt([23,14])
 ItkFormalsListLt(L1(16_1),[23,16])
 ItkFormalsListRt(L1(16_1),[23,16])
 ItkResultTypeAbsent([25,0])
 ItkRaisesSetLt(L0(16_0),[25,0])
 ItkRaisesSetRt(L0(16_0),[25,0])
 ItkSignatureProperRt([23,14])
 ItkDeclScopeRt(L4(16_4))
 ItkProcDeclRt(Id9"P",[25,0])
 ItkOpenDeclListSep(L1(16_1),[20,2])
 ItkProcDeclLt(Id9"P",[25,2])
 ItkDeclScopeLt(L5(16_5))
 ItkSignatureProperLt([25,14])
 ItkFormalsListLt(L1(16_1),[25,16])
 ItkFormalsListRt(L1(16_1),[25,16])
 ItkResultTypeAbsent([26,0])
 ItkRaisesSetLt(L0(16_0),[26,0])
 ItkRaisesSetRt(L0(16_0),[26,0])
 ItkSignatureProperRt([25,14])
 ItkDeclScopeRt(L5(16_5))
 ItkProcDeclRt(Id9"P",[26,0])
 ItkOpenDeclListSep(L1(16_1),[20,2])
 ItkProcDeclLt(Id9"P",[26,2])
 ItkDeclScopeLt(L6(16_6))
 ItkSignatureProperLt([26,14])
 ItkFormalsListLt(L1(16_1),[26,16])
 ItkROFormalLt([26,16])
 ItkROFormalIdListLt(L1(16_1),[26,25])
 ItkDeclNo(D10,[26,25])
 ItkROFormalIdListRt(L1(16_1),[26,25])
 ItkROFormalType([26,28])
 ItkROFormalValue([26,32])
 ItkDeclValAbsent([26,32])
 ItkROFormalRt([26,16])
 ItkFormalsListRt(L1(16_1),[26,16])
 ItkResultTypeAbsent([27,0])
 ItkRaisesSetLt(L0(16_0),[27,0])
 ItkRaisesSetRt(L0(16_0),[27,0])
 ItkSignatureProperRt([26,14])
 ItkDeclScopeRt(L6(16_6))
 ItkProcDeclRt(Id9"P",[27,0])
 ItkOpenDeclListSep(L1(16_1),[20,2])
 ItkProcDeclLt(Id9"P",[27,2])
 ItkDeclScopeLt(L7(16_7))
 ItkSignatureProperLt([27,14])
 ItkFormalsListLt(L1(16_1),[27,16])
 ItkVALUEFormalLt([27,16])
 ItkVALUEFormalIdListLt(L1(16_1),[27,22])
 ItkDeclNo(D11,[27,22])
 ItkVALUEFormalIdListRt(L1(16_1),[27,22])
 ItkVALUEFormalType([27,25])
 ItkDeclTypeAbsent([27,16])
 ItkVALUEFormalValue([27,25])
 ItkVALUEFormalRt([27,16])
 ItkFormalsListRt(L1(16_1),[27,16])
 ItkResultTypeAbsent([29,0])
 ItkRaisesSetLt(L0(16_0),[29,0])
 ItkRaisesSetRt(L0(16_0),[29,0])
 ItkSignatureProperRt([27,14])
 ItkDeclScopeRt(L7(16_7))
 ItkProcDeclRt(Id9"P",[29,0])
 ItkOpenDeclListSep(L1(16_1),[20,2])
 ItkProcDefLt(Id13"Q",[29,2])
 ItkDeclNo(D13,[29,12])
 ItkDeclScopeLt(L8(16_8))
 ItkSignatureProperLt([29,14])
 ItkFormalsListLt(L0(16_0),[29,16])
 ItkFormalsListRt(L0(16_0),[29,16])
 ItkResultTypeAbsent([29,18])
 ItkRaisesSetLt(L0(16_0),[29,18])
 ItkRaisesSetRt(L0(16_0),[29,18])
 ItkSignatureProperRt([29,14])
 ItkProcDefBody([29,18])
 ItkDeclScopeRt(L8(16_8))
 ItkOpenScopeLt(L8(16_8))
 ItkDeclScopeLt(L9(16_9))
 ItkOpenScopeLt(L9(16_9))
 ItkOpenDeclListLt(L0(16_0),[29,20])
 ItkOpenDeclListRt(L0(16_0),[29,20])
 ItkStmtListLt(L0(16_0),[29,26])
 ItkStmtListRt(L0(16_0),[29,26])
 ItkOpenScopeRt(L9(16_9))
 ItkDeclScopeRt(L9(16_9))
 ItkOpenScopeRt(L8(16_8))
 ItkProcDefRt(Id13"Q",[29,30])
 ItkOpenDeclListRt(L2(16_2),[20,2])
 ItkModuleBegin(L1(16_1),[31,2])
 ItkStmtListLt(L0(16_0),[32,2])
 ItkStmtListRt(L0(16_0),[32,2])
 ItkOpenScopeRt(L1(16_1))
 ItkDeclScopeRt(L1(16_1))
 ItkModuleRt(L1(16_1),[11,0])
 ItkRightEnd()
 ItkBOF()
     0
