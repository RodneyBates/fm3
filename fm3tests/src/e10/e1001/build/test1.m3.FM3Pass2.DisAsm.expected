 ItkEOF()
 ItkLeftEnd()
 ItkModuleLt(L2(16_2),[11,0])
 ItkDeclScopeLt(L1(16_1))
 ItkOpenScopeLt(L1(16_1))
 ItkOpenDeclListLt(L6(16_6),[20,2])
 ItkProcDeclLt(Id9"P",[20,2])
 ItkDeclScopeLt(L2(16_2))
 ItkSignatureProperLt([20,14])
 ItkFormalsListLt(L1(16_1),[20,16])
 ItkFormalsListRt(L1(16_1),[20,16])
 ItkResultTypeAbsent([21,0])
 ItkRaisesSetLt(L0(16_0),[21,0])
 ItkRaisesSetRt(L0(16_0),[21,0])
 ItkSignatureProperRt([20,14])
 ItkDeclScopeRt(L2(16_2))
 ItkProcDeclRt(Id9"P",[20,2])
 ItkOpenDeclListSep(L1(16_1),[20,2])
 ItkProcDeclLt(Id9"P",[21,2])
 ItkDeclScopeLt(L3(16_3))
 ItkSignatureProperLt([21,14])
 ItkFormalsListLt(L1(16_1),[21,16])
 ItkVARFormalLt([21,16])
 ItkVARFormalIdListLt(L1(16_1),[21,20])
 ItkDeclNo(D7,[21,20])
 ItkVARFormalIdListRt(L1(16_1),[21,20])
 ItkVARFormalType([21,23])
 ItkVARFormalValue([21,27])
 ItkDeclValAbsent([21,16])
 ItkVARFormalRt([21,16])
 ItkFormalsListRt(L1(16_1),[21,16])
 ItkResultTypeAbsent([23,0])
 ItkRaisesSetLt(L0(16_0),[23,0])
 ItkRaisesSetRt(L0(16_0),[23,0])
 ItkSignatureProperRt([21,14])
 ItkDeclScopeRt(L3(16_3))
 ItkProcDeclRt(Id9"P",[21,2])
 ItkOpenDeclListSep(L2(16_2),[20,2])
 ItkProcDeclLt(Id9"P",[23,2])
 ItkDeclScopeLt(L4(16_4))
 ItkSignatureProperLt([23,14])
 ItkFormalsListLt(L1(16_1),[23,16])
 ItkFormalsListRt(L1(16_1),[23,16])
 ItkResultTypeAbsent([25,0])
 ItkRaisesSetLt(L0(16_0),[25,0])
 ItkRaisesSetRt(L0(16_0),[25,0])
 ItkSignatureProperRt([23,14])
 ItkDeclScopeRt(L4(16_4))
 ItkProcDeclRt(Id9"P",[23,2])
 ItkOpenDeclListSep(L3(16_3),[20,2])
 ItkProcDeclLt(Id9"P",[25,2])
 ItkDeclScopeLt(L5(16_5))
 ItkSignatureProperLt([25,14])
 ItkFormalsListLt(L1(16_1),[25,16])
 ItkFormalsListRt(L1(16_1),[25,16])
 ItkResultTypeAbsent([26,0])
 ItkRaisesSetLt(L0(16_0),[26,0])
 ItkRaisesSetRt(L0(16_0),[26,0])
 ItkSignatureProperRt([25,14])
 ItkDeclScopeRt(L5(16_5))
 ItkProcDeclRt(Id9"P",[25,2])
 ItkOpenDeclListSep(L4(16_4),[20,2])
 ItkProcDeclLt(Id9"P",[26,2])
 ItkDeclScopeLt(L6(16_6))
 ItkSignatureProperLt([26,14])
 ItkFormalsListLt(L1(16_1),[26,16])
 ItkROFormalLt([26,16])
 ItkROFormalIdListLt(L1(16_1),[26,25])
 ItkDeclNo(D10,[26,25])
 ItkROFormalIdListRt(L1(16_1),[26,25])
 ItkROFormalType([26,28])
 ItkROFormalValue([26,32])
 ItkDeclValAbsent([26,16])
 ItkROFormalRt([26,16])
 ItkFormalsListRt(L1(16_1),[26,16])
 ItkResultTypeAbsent([27,0])
 ItkRaisesSetLt(L0(16_0),[27,0])
 ItkRaisesSetRt(L0(16_0),[27,0])
 ItkSignatureProperRt([26,14])
 ItkDeclScopeRt(L6(16_6))
 ItkProcDeclRt(Id9"P",[26,2])
 ItkOpenDeclListSep(L5(16_5),[20,2])
 ItkProcDeclLt(Id9"P",[27,2])
 ItkDeclScopeLt(L7(16_7))
 ItkSignatureProperLt([27,14])
 ItkFormalsListLt(L1(16_1),[27,16])
 ItkVALUEFormalLt([27,16])
 ItkVALUEFormalIdListLt(L1(16_1),[27,22])
 ItkDeclNo(D11,[27,22])
 ItkVALUEFormalIdListRt(L1(16_1),[27,22])
 ItkVALUEFormalType([27,25])
 ItkDeclTypeAbsent([27,16])
 ItkVALUEFormalValue([27,25])
 ItkVALUEFormalRt([27,16])
 ItkFormalsListRt(L1(16_1),[27,16])
 ItkResultTypeAbsent([29,0])
 ItkRaisesSetLt(L0(16_0),[29,0])
 ItkRaisesSetRt(L0(16_0),[29,0])
 ItkSignatureProperRt([27,14])
 ItkDeclScopeRt(L7(16_7))
 ItkProcDeclRt(Id9"P",[27,2])
 ItkOpenDeclListRt(L6(16_6),[20,2])
 ItkModuleBegin(L2(16_2),[29,2])
 ItkStmtListLt(L0(16_0),[30,2])
 ItkStmtListRt(L0(16_0),[30,2])
 ItkOpenScopeRt(L1(16_1))
 ItkDeclScopeRt(L1(16_1))
 ItkModuleRt(L2(16_2),[11,0])
 ItkRightEnd()
 ItkBOF()
     0
